src/OpenTK.Windowing.Desktop/NativeWindow.cs

```diff
@@ -362,11 +371,7 @@ public unsafe Vector2i Location
         public unsafe Vector2i Size
         {
             get => _size;
-            set
-            {
-                _size = value;
-                GLFW.SetWindowSize(WindowPtr, value.X, value.Y);
-            }
```

Comment:
Setting `_size` here is actually not correct. This is handled by `HandleResize` which is called in the [`OnResize`](https://github.com/opentk/opentk/blame/master/src/OpenTK.Windowing.Desktop/NativeWindow.cs#L1155) method.
This change was originally done in this PR https://github.com/opentk/opentk/pull/1076.
Removing the `_size = ...` will make it so this requires the user to correctly call `base.OnResize` when overriding `OnResize` which might be something we may or may not want to do.